contributing-guidelines/template.md

```diff
@@ -372,7 +372,7 @@ You can see an example of buttons in action at the [Intune docs](https://learn.m
 - [Windows](../docs/core/tutorials/using-on-windows.md)
 ```
 
-You can see an example of selectors in action at the [Intune docs](https://learn.microsoft.com/intune/deploy-use/what-to-tell-your-end-users-about-using-microsoft-intune#how-your-end-users-get-their-apps).
+You can see an example of selectors in action in the [Intune documentation](https://learn.microsoft.com/intune/deploy-use/what-to-tell-your-end-users-about-using-microsoft-intune#how-your-end-users-get-their-apps).
```

Comment:
Link does not work, adding to broken link tab. Perhaps try https://learn.microsoft.com/mem/intune/fundamentals/end-user-apps-android